docs/reference/lang/codelab/schema.md

```diff
@@ -604,6 +604,12 @@ mixin PersistentVolumeClaimMixin for PVCProtocol:
         }
 ```
 
+> Note: for the `k8s.api.core.v1` import to work, we need to initialize a module and add the `k8s` module as a dependency:
```

Comment:
This is the first time in this doc where external dependencies are used. Perhaps it's worth mentioning how to run this example.